app/models/registrar.rb

```diff
@@ -0,0 +1,11 @@
+class Registrar < ApplicationRecord
+  belongs_to :user
```

Comment:
I know this isn't open for review officially, but I didn't notice that and looked so here's my feedback unofficially :)

- I don't think we want a user associated with these files unless someone else has provide use cases that contradict this. Were you thinking of a use case I may not be? I'm assuming we'll just process these and not care who uploaded them because only admins can upload them. I guess maybe keeping them around after processing for a period of time and knowing who did stuff isn't awful though so... maybe your way is actually better.
- We will need permissions added in ability.rb to ensure only thesis_admins can upload files.

I do think we need to have permissions added to ability

Reply:
Yeah, I've gone back and forth with myself about whether we need the user information or not. I tried doing without it initially, but ran into some issues that seemed to be forcing me into having this. I may not have looked closely enough, though. The current requirement doesn't list a user field, so I suspect that this is the intended path.